experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1531,6 +1536,79 @@ def _validate_bucket_duplicates(self, data):
 
         return data
 
+    def _validate_localization_content(self, data):
+        is_localized = data.get("is_localized")
+
+        if not is_localized:
+            return data
+
+        localized_content = data.get("localized_content")
+        locales = data.get("locales")
+        min_version = data.get("firefox_min_version", "")
+        if min_version == "" or (
+            NimbusExperiment.Version.parse(min_version)
+            < NimbusExperiment.Version.parse(NimbusExperiment.Version.FIREFOX_113)
```

Comment:
we should move this to our constants file, so we know which minimum version it supports and can refer that variable here, for example https://github.com/mozilla/experimenter/blob/fdff151a9a77303f588efbb396f98f3f2c3fb494/experimenter/experimenter/experiments/constants.py#L345